packages/webview_flutter/example/android/app/src/androidTest/java/io/flutter/plugins/webviewflutterexample/WebViewTest.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.webviewflutterexample;
+
+import static org.junit.Assert.assertTrue;
+
+import androidx.test.core.app.ActivityScenario;
+import io.flutter.plugins.webviewflutter.WebViewFlutterPlugin;
+import org.junit.Test;
+
+public class WebViewTest {
```

Comment:
what runner is this using? do you want to use AndroidJUnit4?

Reply:
From my understanding of Gradle, it uses `AndroidJUnit4` as default because of https://github.com/flutter/plugins/blob/master/packages/webview_flutter/example/android/app/build.gradle#L41

Reply:
Also explained here https://developer.android.com/training/testing/junit-runner#ato-gradle

Reply:
sgtm